dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -290,7 +290,7 @@ static void Edit_BackWord(HWND hwndEdit)
     if (iStart != iEnd)
         return;
 
-    DWORD cchText = GetWindowTextLengthW(hwndEdit);
+    INT cchText = GetWindowTextLengthW(hwndEdit);
```

Comment:
It `should` probably never happen, but theoretically cchText can be negative.

f.e. riched20 can return `E_INVALIDARG`, from `WM_GETTEXTLENGTH`, which translates to `0x80070057L`, which will be negative when interpreted as signed int.

Reply:
Made `cchText` a `size_t`. Added `iStart` and `iEnd` sign checks. OK.